subsys/bluetooth/controller/hci_internal.h

```diff
@@ -24,6 +27,43 @@
  */
 int hci_internal_cmd_put(uint8_t *cmd_in);
 
+/** A user implementable HCI command handler
+ *
+ * What is done in the command handler is up to the user.
+ * Parameters can be returned to the host through the raw_event_out output parameter.
+ *
+ * When the command handler returns a Command Complete event is generated, unless
+ * the opcode is known to generate a Command Status event.
```

Comment:
As I see it, it is not possible to add a new command that returns a Command Status. Should we add a new boolean output to the handler that defines if a Command Complete or a Command Status should be generated?

Reply:
Done